### DISCUSSION TITLE
Whistleblower Policy

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#37] [Added]
### Type name

Whistleblower Policy

### Alternative type names

Whistleblower Protections

### Writer

service provider

### Audience

employees

### Object

reporting on suspected misconduct and illegal acts and prevention of retaliation

### Examples

| Example | Service Name | URL | Title | Jurisdiction(s) |
|:----|:----|:----|:------|:-------------|
| ~1~ | ~DNSFilter~ | ~[URL 🔗](https://www.dnsfilter.com/whistleblower-policy)~ | ~WHISTLEBLOWER POLICY~ | - |
| 2 | GOV.UK | [URL 🔗](https://www.gov.uk/whistleblowing) | Whistleblowing for employees | - |
| 3 | University of California | [URL 🔗](https://policy.ucop.edu/doc/1100171/Whistleblower) | Whistleblower Policy | - |
| 4 | Nottinghamshire County Council | [URL 🔗](https://www.nottinghamshire.gov.uk/jobs-and-working/working-for-us/whistleblowing-policy) | Whistleblowing Policy | - |
| 5 | U.S Department of Labor | [URL 🔗](https://www.dol.gov/general/topics/whistleblower) | Whistleblower Protections | - |
| 6 | Air Liquide | [URL 🔗](https://www.airliquide.com/sites/airliquide.com/files/2023-01/whistleblowing-policy.pdf) | Whistleblowing Policy | 🌐 |
| 7 | ING | [URL 🔗](https://www.ing.com/MediaEditPage/ING-Whistleblower-policy.htm) | Global Whistleblower Policy | 🌐 |
| 8 | Ministère de l'Europe et des Affaires étrangères | [URL 🔗](https://www.diplomatie.gouv.fr/fr/le-ministere-et-son-reseau/lanceurs-d-alerte/) | Lanceurs d’alerte | 🇫🇷 |
| 9 | Office national de sécurité sociale | [URL 🔗](https://www.onss.be/politique-relative-a-la-protection-des-lanceurs-dalerte) | Politique relative à la protection des lanceurs d’alerte | 🇧🇪 |
| 10 | ICCROM | [URL 🔗](https://www.iccrom.org/sites/default/files/03_eng_ICCROM_Policy_Whistleblower_2020_final.pdf) | Whistleblower and Anti-retaliation Policy | 🇪🇺 |
| 11 | Whirlpool | [URL 🔗](https://www.whirlpool.fr/api/dataentities/AT/documents/8210b9b6-a2df-11ed-83ab-0e147c68383d/file/attachments/8210b9b6-a2df-11ed-83ab-0e147c68383d_feed_file.pdf) | POLITIQUE FRANCAISE RELATIVE AUX LANCEURS D’ALERTE | 🇫🇷 |
| 12 | Pierre Bleue Belge | [URL 🔗](https://www.pierrebleuebelge.be/politique-lanceurs-dalerte/) | POLITIQUE LANCEURS D’ALERTE | 🇧🇪 |

### References

- [Loi n°2016-1691 du 9 décembre 2016 relative à la transparence, à la lutte contre la corruption et à la modernisation de la vie économique, dite « Sapin II »](https://www.legifrance.gouv.fr/loda/id/JORFTEXT000033558528)
- [Directive (EU) 2019/1937 of the European Parliament and of the Council of 23 October 2019 on the protection of persons who report breaches of Union law](https://eur-lex.europa.eu/legal-content/en/TXT/?uri=CELEX%3A32019L1937)
- [Whistleblower Protection Act of 1989](https://www.govinfo.gov/content/pkg/STATUTE-103/pdf/STATUTE-103-Pg16.pdf)

**Comment by clementbiron** [comment]
Additional references:
- [Directive (EU) 2019/1937 of the European Parliament and of the Council of 23 October 2019 on the protection of persons who report breaches of Union law](https://eur-lex.europa.eu/legal-content/en/TXT/?uri=CELEX%3A32019L1937)
- [103 STAT. 16 PUBLIC LAW 101-12—APR. 10, 1989](https://www.govinfo.gov/content/pkg/STATUTE-103/pdf/STATUTE-103-Pg16.pdf)

**Comment by MattiSG** [comment]
- Example 1 is not available anymore
- Examples 2, 4 and 5 are from public institutions. It is not clear to me if in example 2 the “service” is “GOV.UK” or the government of the United Kingdom.

Additional examples:

- Ministère de l'Europe et des Affaires étrangères | https://www.diplomatie.gouv.fr/fr/le-ministere-et-son-reseau/lanceurs-d-alerte/ | Lanceurs d’alerte | 🇫🇷
- Office national de sécurité sociale | https://www.onss.be/politique-relative-a-la-protection-des-lanceurs-dalerte | Politique relative à la protection des lanceurs d’alerte | 🇧🇪
- ICCROM | https://www.iccrom.org/sites/default/files/03_eng_ICCROM_Policy_Whistleblower_2020_final.pdf | Whistleblower and Anti-retaliation Policy | 🇪🇺
- Whirlpool | https://www.whirlpool.fr/api/dataentities/AT/documents/8210b9b6-a2df-11ed-83ab-0e147c68383d/file/attachments/8210b9b6-a2df-11ed-83ab-0e147c68383d_feed_file.pdf | POLITIQUE FRANCAISE RELATIVE AUX LANCEURS D’ALERTE | 🇫🇷
- Pierre Bleue Belge | https://www.pierrebleuebelge.be/politique-lanceurs-dalerte/ | POLITIQUE LANCEURS D’ALERTE | 🇧🇪

In the case of Whirlpool, they also have a global [Integrity manual](https://www.whirlpoolcorp.com/our-impact/governance-and-policies/integrity-manual/), but only the French version seems to be specific to whistleblowing, due to the Sapin II law.
Similarly, I can find the French version of a whistleblower policy for Pierre Bleue Belge, but not an English version, even though their website is otherwise translated in English and Dutch.

## References

- [Loi n°2016-1691 du 9 décembre 2016 relative à la transparence, à la lutte contre la corruption et à la modernisation de la vie économique, dite « Sapin II »](https://www.legifrance.gouv.fr/loda/id/JORFTEXT000033558528)
